docs/changes.rst

```diff
@@ -187,6 +187,9 @@ Released: not yet
 * Removed a circumvention for a pywbem bug related to colons in WBEM URIs
   that was fixed in pywbem 0.13.0. (See issue #131)
 
+* Corrected issue with use-pull general option that causes issues with using
+  the 'either' option with servers that do not have pull. (See issue #530)
```

Comment:
Add statement that this adds a "use_pull" argument to PywbemServer() (it is incompatible to add that in the middle).

Reply:
DONE

Reply:
NOTE: removed duplicated comment about issue #530 (it was in bugs and enhancements) and added one for this pr.